docs/theme/power2lines.rst

```diff
@@ -0,0 +1,44 @@
+.. _power2lines:
```

Comment:
You shouldn't need to add anchors to the start of documents. You can link to them by the 
```
:doc:`filename`
```
syntax.